spec/requests/users_spec.rb

```diff
@@ -2,36 +2,32 @@
 
```

Comment:
- [ ] It would be better if all our test cases marks `green` we have 2 pending tests that needs to be addressed :1st_place_medal: 
![Screenshot from 2022-09-27 10-25-55](https://user-images.githubusercontent.com/46388113/192502659-d464472f-f974-405b-a185-24e50e1ae07d.png)